app/src/pages/common/membersPage/membersPage.jsx

```diff
@@ -122,7 +120,10 @@ export class MembersPage extends Component {
     }
     return (
       !!filter && (
-        <NoResultsForFilter filter={this.props.filter} notFoundMessage={messages.membersNotFound} />
+        <NoResultsForFilter
+          filter={this.props.filter}
+          notFoundMessage={this.props.intl.formatMessage(messages.membersNotFound)}
```

Comment:
Why do you use `formatMessage` here? I see that this component format messages by itself.

Reply:
Done.